translate/storage/xliff.py

```diff
@@ -526,7 +526,7 @@ def getlocations(self):
                 if sourceFile is not None and lineNumber is not None:
                     locations.append(sourceFile[1] + ":" + lineNumber[1])
 
-        return ", ".join(x for x in locations if x is not None)
+        return locations
```

Comment:
Hello. 
locations is set to [] at the beginning of the function, then it will return an empty array if no locations are found in the for loop.

Can you elaborate a bit more please on the _None_ scenario? Thanks

Reply:
I mean that `None` should not be returned as a location. You add filtering of this in the test, but I think it should be here instead:

```suggestion
        return [x for x in locations if x is not None]
```

Of if there is no `None` in the list, the filtering should be removed from the testcase.